.github/workflows/github-add-label.yaml

```
@@ -0,0 +1,60 @@
+name: github-add-label
+
+on:
+  workflow_call:
+    inputs:
+      PRIMUS_REF:
+        description: 'The primus ref to checkout.'
+        required: true
+        default: 'main'
```

actionlint (1.7.4)

input "PRIMUS_REF" of workflow_call event has the default value "main", but it is also required. if an input is marked as required, its default value will never be used

(events)